### DISCUSSION TITLE
Triggering workflow only for Pull Request Comment

### DISCUSSION THREAD

**Discussion by ibakshay** [community/community#25531] [Actions]
<p>Hello,</p>
<p>Is there  a way in triggering the workflow only for the Pull Request Comments  and not for the issue comment ?. <br>
I am aware that on  <strong>issue_comment </strong> will trigger the workflow for both pull request comment as well as issue comment, But I don’t want to trigger the workflow at all for issue comments.</p>
<p>It will be great If anyone can help me out on this. </p>

**Comment by BrightRan** [comment, marked as answer]
<p>Currently, there is not any easy way to set filter on Issue comment event to let workflow be triggered only when comment on PR or only on issue.</p>
<p>However, from the <strong>github context</strong> for the event, I found some different between issue and PR.</p>
<ul>
<li>If comment on PR triggers the workflow, the value of <strong>event.comment.html_url</strong> is:</li>
</ul>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2F%253corg%253e%2F%253crepo%253e%2Fpull%2F%253cpr_id%253e%23issuecomment-%3Ccomment_id&amp;data=02%7C01%7Cv-brira%40microsoft.com%7Cde28ae9f7dde45a9d03e08d77d6420d9%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637115741309103763&amp;sdata=ID7WkFHQ%2BY%2FeCJNLK8HkO0HPlNrMg1%2B0UM1AA0LM9ag%3D&amp;reserved=0" rel="nofollow noopener">https://github.com/&lt;org&gt;/&lt;repo&gt;/pull/&lt;pr_id&gt;#issuecomment-&lt;comment_id</a>&gt;</p>
<ul>
<li>If comment on issue triggers the workflow, the value of <strong>event.comment.html_url</strong> is:</li>
</ul>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2F%253corg%253e%2F%253crepo%253e%2Fissues%2F%253cissue_id%253e%23issuecomment-%3Ccomment_id&amp;data=02%7C01%7Cv-brira%40microsoft.com%7Cde28ae9f7dde45a9d03e08d77d6420d9%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637115741309113757&amp;sdata=BsDeuDHkUWc7pt9Uckr961nPHN0fY7pze3%2BeHSO4XY4%3D&amp;reserved=0" rel="nofollow noopener">https://github.com/&lt;org&gt;/&lt;repo&gt;/issues/&lt;issue_id&gt;#issuecomment-&lt;comment_id</a>&gt;</p>
<p>So you can try the below <strong>if conditional</strong> to skip all the jobs when the workflow is triggered by comment on issue.</p>
<pre><code>jobs:
 job01:
    if: contains(github.event.comment.html_url, '/pull/')
</code></pre>

**Comment by ibakshay** [comment]
<p>Thank you very much. It certainly helped. However, Filtering with jobs are failing and I am not able to see the runner logs too. Could you please check this issue ?</p>
